### PR TITLE
Add BALSAMIC to list of valid workflows to upload

### PR DESCRIPTION
**Pull request by hassanfa**
This PR adds BALSAMIC to Vogue upload.

This is due to issue #925 . We'll fix that on Vogue side on this PR: https://github.com/Clinical-Genomics/vogue/pull/212

### How to prepare for test
- [ ] ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] paxa the environment: `paxa`
- [ ] install on stage (example for hasta):
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-cg-stage.sh fix/workflow_name_vogue_upload_i`

### How to test
- [ ] do ...

### Expected test outcome
- [ ] check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review
- [x] code approved by HS
- [x] tests executed by HFA
- [x] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan
- [ ] Document in ...
- [ ] Deploy this branch
- [ ] Inform to ...

### CODE REVIEW

File: cg/cli/upload/vogue.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 LOG = logging.getLogger(__name__)
 
-VOGUE_VALID_BIOINFO = [str(Pipeline.MIP_DNA)]
```
Comment (henrikstranneheim) [inline cg/cli/upload/vogue.py:19]: But should we not allow both production pipelines i.e. MIP-DNA, and Balsamic?

Comment (hassanfa) [reply cg/cli/upload/vogue.py:19]: yes, I'm testing for now. needed a working branch to test on Stage :-) I'm fixing it on vogue side in parallel as well.

**Comment by hassanfa** [comment]
Upload for mip is working:

image

**Comment by hassanfa** [comment]
Vogue shows BALSAMIC results loaded:
image

**Comment by hassanfa** [comment]
Hardest code I have ever written! Should be a quick review!

**Comment by hassanfa** [comment]
Bump

**Review by henrikstranneheim — APPROVED** [review]
👍

**Comment by hassanfa** [comment]
Bump

**Comment by Mropat** [comment]
yes yes yes

**Comment by Mropat** [comment]
Installed on prod

```
Building wheels for collected packages: cg
  Running setup.py bdist_wheel for cg ... done
  Stored in directory: /home/maryia.ropart/tmp/pip-ephem-wheel-cache-cz5m5crl/wheels/5c/23/37/e3635140fad58f81760c6db5705a64683daf19230a7bb26dda
Successfully built cg
Installing collected packages: cg
  Found existing installation: cg 18.9.0
    Uninstalling cg-18.9.0:
      Successfully uninstalled cg-18.9.0
Successfully installed cg-18.9.1

```

**Pull request merged**